src/Output/ListFormatter.php

```diff
@@ -8,9 +8,9 @@
 
 class ListFormatter implements OutputFormatter
 {
-    const ROW_TYPE_EMPTY   = 1;
-    const ROW_TYPE_COMMENT = 2;
-    const ROW_TYPE_COMMAND = 4;
+    protected const ROW_TYPE_EMPTY   = 1;
+    protected const ROW_TYPE_COMMENT = 2;
+    protected const ROW_TYPE_COMMAND = 4;
```

Comment:
To fix following warning message, adding the `protected` to be visible these const values:

```
FILE: /home/lee/crontab/src/Output/ListFormatter.php
--------------------------------------------------------------------------------
FOUND 0 ERRORS AND 3 WARNINGS AFFECTING 3 LINES
--------------------------------------------------------------------------------
 11 | WARNING | Visibility must be declared on all constants if your project
    |         | supports PHP 7.1 or later
 12 | WARNING | Visibility must be declared on all constants if your project
    |         | supports PHP 7.1 or later
 13 | WARNING | Visibility must be declared on all constants if your project
    |         | supports PHP 7.1 or later
--------------------------------------------------------------------------------
```